django/views/i18n.py

```diff
@@ -284,6 +287,22 @@ def get_javascript_catalog(locale, domain, packages):
     return catalog, plural
 
 
+def _get_locale(request):
+    if request.GET and LANGUAGE_QUERY_PARAMETER in request.GET:
```

Comment:
Right. Given the original check for ``if request.GET`` is almost 10 years old dating back to commit 5917fdc, we may safely assume there's no reason to leave it like that.